lib/User_Proxy.php

```diff
@@ -363,4 +369,18 @@ public function getUserName($uid) {
 		}
 		return $result;
 	}
+
+	public function testConnection(){
+		foreach ($this->serverConfigPrefixes as $configPrefix) {
```

Comment:
We should rename it to "checkForConnectionErrors" or "checkConnectionErrorsOccured"